core/src/queue.rs

```diff
@@ -424,18 +418,44 @@ mod tests {
     }
 
     #[test]
+    #[ignore = "Multisignature is not working for now. See #2595"]
     fn push_multisignature_tx() {
```

Comment:
This test wasn't test multisignatures at all previously, because in this test we are putting transaction that already pass signature check and then we are adding another signature to it.